test/0.0.1-rc.39/Competition.spec.ts

```diff
@@ -289,6 +290,7 @@ describe('Competition', () => {
             url: proposalUrl,
             tags: tagsList,
             suggester: accounts[0].address.toLowerCase(),
+            beneficiary: accounts[0].address.toLowerCase(),
```

Comment:
the code seems fine,but it would be good to have a test where "suggestor !== beneficiary" just to make sure

Reply:
Added